src/pages/TabsPage/TabsPage.tsx

```diff
@@ -0,0 +1,24 @@
+import { useParams } from 'react-router-dom';
+import { Tabs } from '../../components/Tabs/Tabs';
+import { Tab } from '../../types/Tab';
+
+export const tabs: Tab[] = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
+
+export const TabsPage: React.FC = () => {
+  const { tabId = '0' } = useParams();
```

Comment:
Why do you need to use the default `0`? You can use `undefined` and create `selectedTabId` as not required because you will never use the `0` value.

Reply:
ok. thanks